scripts/ops/reclaim_supabase_swap.sql

```
@@ -0,0 +1,95 @@
+--    Requires every expected legacy entry to exist in public.manifest.
+--    Re-run until notice shows updated=0.
+-- ---------------------------------------------------------------------------
+SELECT public.null_migrated_app_version_manifests();
+
+VACUUM (VERBOSE) public.app_versions;
+-- Routine VACUUM does not shrink TOAST. After nulling is done, compact in the
+-- maintenance window (exclusive lock):
+-- VACUUM (FULL, VERBOSE) public.app_versions;
+
+-- ---------------------------------------------------------------------------
+-- 4) Trim audit_logs older than 30 days (bounded batches). Re-run until deleted=0.
```

SQLFluff (4.2.2)

[error] Line is too long (83 > 80).

(LT05)